src/doc/faq.dox

```diff
@@ -45,6 +45,40 @@ namespace kaldi {
   - Speech Recognition with Weighted Finite-State Transducers for WSFT
   - A Bit of Progress in Language Modeling (Extended Version)
   
+  For those who may want a "Kaldi Book" with turorial on theory and implementation like what HTK Book does, we would generally just say sorry. As Dan explains in this <a href=https://groups.google.com/forum/#!topic/kaldi-help/3LBSzmploC0> post </a>, the field of speech recognition is moving so fast that we need to implement too many things in Kaldi and have no time to write such a book.
```

Comment:
turorial -> tutorial

Reply:
Thanks jtrmal for the comments. I have fixed the typos. I'll check the spellings in the following PR with grammar check tool. Thanks in advance for your review.